forms/gridfield/GridFieldExportButton.php

```diff
@@ -111,7 +111,11 @@ public function generateExportFileData($gridField) {
 			// determine the CSV headers. If a field is callable (e.g. anonymous function) then use the
 			// source name as the header instead
 			foreach($csvColumns as $columnSource => $columnHeader) {
-				$headers[] = (!is_string($columnHeader) && is_callable($columnHeader)) ? $columnSource : $columnHeader;
+				if (is_array($columnHeader) && array_key_exists('title', $columnHeader)) { 
```

Comment:
Weird... Those are unicode line separators (U+2028) but I can't find out why those are being used... I've will remove those and squash the commits onto a new branch.